pom.xml

```diff
@@ -92,7 +92,7 @@
     <github.global.server>github</github.global.server>
     <google.auth.version>0.6.0</google.auth.version>
     <grpc.version>1.0.3</grpc.version>
-    <gax.version>0.1.3</gax.version>
+    <gax.version>0.1.5</gax.version>
```

Comment:
Since the bundling stuff is breaking, could we release the next gax at 0.2.0 instead?